lib/index.js

```diff
@@ -23,7 +25,15 @@ var Castle = module.exports = integration('Castle')
 Castle.prototype.initialize = function() {
   window._castle = window._castle || {};
   window._castle.q = window._castle.q || [];
-  window._castle.q.push(['setAppId', this.options.appId]);
+  window._castle.q.push(['setKey', this.options.publishableKey]);
+
+  if (this.options.cookieDomain) {
+    window._castle.q.push(['setCookieDomain', this.options.cookieDomain]);
+  }
+
+  if (this.options.autoPageview === false) {
+    window._castle.q.push(['autoTrack', this.options.autoPageview]);
```

Comment:
@hankim813 in `castle.js` you have to opt out of the autotracking. So this should stay the same. Just to double check. The way this integration is implemented now, we should always get at least on `page` call, correct?